docs/_config.yml

```
@@ -0,0 +1,22 @@
+title: Frequency
+description: "Offline, privacy-first voice rooms over Bluetooth LE — no internet required."
+theme: minima
+
+minima:
+  date_format: "%B %-d, %Y"
+  social_links:
+    - { platform: github, user_url: "https://github.com/ElodinLaarz/walkie-talkie" }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)